spec/factories/votes.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :vote do
+    user { nil }
```

Comment:
It's worth initialising the factory with a user, so you can just leave this line as `user`.